test/integration/matchers/toBeShape/index.ts

```diff
@@ -66,18 +66,26 @@ export function buildExpectedShapeCellStyle(expectedModel: ExpectedShapeModelEle
   style.align = expectedModel.align ?? 'center';
   style.strokeWidth = style.strokeWidth ?? expectedStrokeWidth(expectedModel.kind);
 
-  style.fillColor =
-    expectedModel.fill?.color ??
-    ([ShapeBpmnElementKind.LANE, ShapeBpmnElementKind.POOL, ShapeBpmnElementKind.TEXT_ANNOTATION, ShapeBpmnElementKind.GROUP].includes(expectedModel.kind)
-      ? 'none'
-      : style.fillColor);
+  const fill = expectedModel.fill;
+  if (fill) {
+    style.fillColor = fill.color ?? style.fillColor;
+    style.fillOpacity = fill.opacity;
+  }
+  if (!fill?.color && [ShapeBpmnElementKind.LANE, ShapeBpmnElementKind.POOL, ShapeBpmnElementKind.TEXT_ANNOTATION, ShapeBpmnElementKind.GROUP].includes(expectedModel.kind)) {
+    style.fillColor = 'none';
+  }
```

Comment:
**question:** this seems to be a refactoring that is not related to the new feature.
If so, please revert and eventually create a dedicated PR.

Reply:
It's a rest of https://github.com/process-analytics/bpmn-visualization-js/pull/2760/commits/bbc9b1e062b70aad2a1877004962f87d1fb124bf.

The old code wasn't clear for me for the real test.
I can move in another PR, but I was lazy 😆